charts/backingservices/values.yaml

```diff
@@ -29,10 +29,18 @@ srs:
     # To use your own Elasticsearch cluster, set srsStorage.provisionInternalESCluster to false and then
     # set the external Elasticsearch cluster URL and port details below when using an externally managed elasticsearch
     # Ensure that the specified endpoint is accessible from the kubernetes cluster pods.
-    # Currently the elasticsearch connection does not support any modes of authentication and should be es endpoint APIs' accessible without authentication.
+    # Currently the elasticsearch connection is supported with Basic Authentication  and AWS IAM Role based Authentication .
 #    domain: managed-elasticsearch.acme.io
 #    port: 443
 #    protocol: https
+    # If externally managed Elasticsearch cluster is authenticated with Basic Authentication then please set below srs.srsStorage.basicAuthentication's srs.srsStorage.basicAuthentication.username and srs.srsStorage.basicAuthentication.password values
```

Comment:
To configure authentication for the externally managed Elasticsearch cluster to use Basic Authentication then uncomment and add the parameters, srs.srsStorage.basicAuthentication.username and srs.srsStorage.basicAuthentication.password

To configure authentication for the externally managed Elasticsearch cluster to use an AWS IAM Role then uncomment and add the parameters, srs.srsStorage.awsIAM and srs.srsStorage.awsIAM.region